spec/io/sql_data_source_spec.rb

```diff
@@ -62,5 +62,25 @@
         }.to raise_error(ArgumentError)
       end
     end
+
+    context 'with path to sqlite3 file' do
+      it 'returns a dataframe' do
+        result = Daru::IO::SqlDataSource.make_dataframe(db_name, query)
+        # result = Daru::DataFrame.from_sql(db_name, query)
+        expect(result).to be_a(Daru::DataFrame)
+        expect(result.nrows).to eq(2)
+        expect(result.row[0][:id]).to eq(1)
+        expect(result.row[0][:name]).to eq('Homer')
+      end
```

Comment:
Just checking in to let you know that I saw this. I'll have a chance to take care of this at some point tomorrow.